Src/HALALMock/Services/DigitalOutputService/DigitalOutputService.cpp

```diff
@@ -24,7 +27,8 @@ void DigitalOutputService::turn_off(uint8_t id){
 	}
 
 	Pin pin = DigitalOutputService::service_ids[id];
-	HAL_GPIO_WritePin(pin.port, pin.gpio_pin, (GPIO_PinState)PinState::OFF);
+	(*gpio_memory + pin_offset[pin.gpio_pin])&= 0xFE;
```

Comment:
```suggestion
	(*gpio_memory + pin_offset[pin.gpio_pin]) = PinState::LOW;
```
don't know the exact enum variant